lib/dynosaur/controllers/papertrail_controller_plugin.rb

```diff
@@ -10,11 +10,11 @@ def initialize(config)
 
         @min_resource = Dynosaur::Addons.plans_for_addon('papertrail').find {|plan| plan['name'] == min_resource_name }
         if @min_resource.nil?
-          raise "Min resource not found with name #{min_resource_name}"
+          raise StandardError.new "Min resource not found with name #{min_resource_name}"
```

Comment:
doesn't `raise` create a `StandardError` by default? or are you just doing that for clarity?